docs/source/quick_start/examples.rst

```diff
@@ -3,9 +3,23 @@
 ##########
```

Comment:
Maybe the following commit message is better:
docs: add a table of examples' details to examples.rst